SourcingTests/FetchedResultsDataProviderTests.swift

```diff
@@ -236,17 +247,26 @@ class FetchedResultsDataProviderTests: XCTestCase {
         dataProvider.controllerDidChangeContent(fetchedResultsController as! NSFetchedResultsController<NSFetchRequestResult>)
         
         //Then
-        XCTAssertEqual(updatesNumber, 2)
+        XCTAssertEqual(updateIndexPath, newIndexPath)
+        XCTAssertEqual(moveIndexPaths, [oldIndexPath, newIndexPath])
     }
     
-    func testProcessUpdatesNumberForOtherThenMoveChange() {
+    func testProcessUpdatesForUpdateChange() {
         //Given
-        var updatesNumber = 0
+        var updateIndexPath = IndexPath()
         
         dataProvider = try! FetchedResultsDataProvider(fetchedResultsController: fetchedResultsController, dataProviderDidUpdate: { _ in })
         dataProvider.whenDataProviderChanged = { updates in
-            if !updates!.isEmpty {
-                updatesNumber += 1
+            if updates!.isEmpty {
+                return
+            }
+           
+            for update in updates! {
```

Comment:
can you replace ` for update in updates!` with ` for update in updates ?? []`. This will cause the test to fail instead of crashing when updates is nil

Reply:
you're right - I will fix it today